config/crd/bases/core.openstack.org_openstackcontrolplanes.yaml

```diff
@@ -15953,6 +15953,23 @@ spec:
                   - type
                   type: object
                 type: array
+              tls:
+                properties:
+                  caBundleSecretName:
+                    type: string
+                  endpoint:
+                    additionalProperties:
+                      properties:
+                        expires:
```

Comment:
should this have `format: date-time`?

Reply:
you mean to format it different? right now its just the NotAfter string from the CA

Reply:
wondering if this should be just a list of CAs, instead of endpoint map. we know that there will be also others

Reply:
IIUC format is also used to be more specific about the type https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.0.md#dataTypes

Reply:
done